content/introduction.md

```diff
@@ -25,10 +25,10 @@ $ nest new project-name
 
 #### Alternatives
 
-Alternatively, to install the TypeScript starter project with **Git**:
+Alternatively, to install the TypeScript starter project with [degit](https://github.com/Rich-Harris/degit):
 
 ```bash
-$ git clone https://github.com/nestjs/typescript-starter.git project
+$ npx degit nestjs/typescript-starter project
 $ cd project
```

Comment:
Maybe instead of replacing the current example, we can just add another code snippet, right below this one explaining that `degit` can be used in order to clone the repository without its commit history etc.?